hack/dev/run-in-minikube.sh

```diff
@@ -16,6 +16,7 @@ $SCRIPT_ROOT/godelw docker build --verbose
 
 kubectl apply -f $SCRIPT_ROOT/examples/namespace.yml
 
-kubectl create configmap scheduler-secrets --namespace=spark --from-file="${SCRIPT_ROOT}/out/tmp/generated_keys/"
+kubectl create configmap scheduler-secrets --namespace=spark --from-file="${SCRIPT_ROOT}/out/generated_keys/"
```

Comment:
nit nit - maybe extract `${SCRIPT_ROOT}/out/generated_keys/` to a shared constant

Reply:
👍